src/components/FilterBar.vue

```diff
@@ -0,0 +1,137 @@
+<script setup lang="ts">
+import { FilterType } from "../types/filter";
+
+const objektClass = defineModel<string | null>("objektClass", {
+  required: true,
+});
+const objektSeason = defineModel<string | null>("objektSeason", {
+  required: true,
+});
+const group = defineModel<string | null>("group", { required: true });
+const artist = defineModel<string | null>("artist", { required: true });
+
+const tripleSMembers = [
+  "SeoYeon",
+  "HyeRin",
+  "JiWoo",
+  "ChaeYeon",
+  "YooYeon",
+  "SooMin",
+  "NaKyoung",
+  "YuBin",
+  "Kaede",
+  "DaHyun",
+  "Kotone",
+  "YeonJi",
+  "Nien",
+  "SoHyun",
+  "Xinyu",
+  "Mayu",
+  "Lynn",
+  "JooBin",
+  "HaYeon",
+  "ShiOn",
+  "ChaeWon",
+  "Sullin",
+  "SeoAh",
+  "JiYeon",
+];
+
+const tripleSSubunits = [
```

Comment:
I'd recommend adding Aria, Visionary Vision and Hatachi (also ACID EYES ?) as those are also subunits

Reply:
Are they in the API and is that the exact way they are written?
Ideally, it would fetch the available option dynamically though, but I think we'd need to provide a custom endpoint for that.